doc/faq.rst

```diff
@@ -52,94 +51,40 @@ For example::
-on the line just above the problem. ``...`` in the following example is short for the
-list of messages you want to disable.
-
-For larger amounts of code, you can add ``#pylint: disable=...`` at the block level
-to disable messages for the entire block. It's possible to re-enable a message for the
-remainder of the block with ``#pylint: enable=...``. A block is either a scope (say a
-function, a module) or a multiline statement (try, finally, if statements, for loops).
-Note: It's currently impossible to `disable inside an else block`_.
-
 Read :ref:`message-control` for details and examples.
 
-.. _`disable inside an else block`: https://github.com/PyCQA/pylint/issues/872
-
-Is there a way to disable a message for a particular module only?
------------------------------------------------------------------
-
-Yes, you can disable or enable (globally disabled) messages at the
-module level by adding the corresponding option in a comment at the
-top of the file: ::
-
-    # pylint: disable=wildcard-import, method-hidden
-    # pylint: enable=too-many-lines
-
-How can I never check a given module?
-----------------------------------------------------
+Pylint gave my code a negative rating out of ten. That can't be right!
+----------------------------------------------------------------------
 
-Add ``#pylint: skip-file`` at the beginning of the module.
+Prior to Pylint 2.13.0, the score formula used by default had no lower
+bound. The new default score formula is ::
 
-In order to ease finding which modules are ignored an Information-level message
-`file-ignored` is emitted.
+    max(0, 0 if fatal else 10.0 - ((float(5 * error + warning + refactor + convention) / statement) * 10))
 
+If your project contains a configuration file created by an earlier version of
+Pylint, you can set ``evaluation`` to the above expression to get the new
+behavior. Likewise, since negative values are still technically supported,
+``evaluation`` can be set to a version of the above expression that does not
+enforce a floor of zero.
 
-How do I avoid getting unused argument warnings when I have no control over received arguments?
------------------------------------------------------------------------------------------------
+How do I avoid getting unused argument warnings for API I do not control?
+-------------------------------------------------------------------------
 
-Prefix (ui) the callback's name by `cb_`, as in cb_onclick(...). By
+Prefix (ui) the callback's name by `cb_` (callback), as in cb_onclick(...). By
 doing so arguments usage won't be checked. Another solution is to
 use one of the names defined in the "dummy-variables" configuration
 variable for unused argument ("_" and "dummy" by default).
 
 What is the format of the configuration file?
 ---------------------------------------------
 
-Pylint uses ConfigParser from the standard library to parse the configuration
```

Comment:
I'm not sure this question is answered by the link. The link is great, but maybe add one sentence about formats accepted.

Reply:
Would that work ? https://github.com/PyCQA/pylint/pull/6726/files#r884167586